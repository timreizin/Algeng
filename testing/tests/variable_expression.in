a=5
a
b=a+2
b
a*b+a
c=a*a*b+a
c
-c+a-b
