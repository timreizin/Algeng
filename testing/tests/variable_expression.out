5
7
40
180
-182
